log-sink/src/test/java/org/springframework/cloud/stream/module/log/LogSinkApplicationTests.java

```diff
@@ -16,27 +16,20 @@
 
 package org.springframework.cloud.stream.module.log;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertSame;
-import static org.mockito.Mockito.spy;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-
-import java.util.Properties;
+import static org.junit.Assert.*;
```

Comment:
I believe we prefer not to have wildcard imports... can import the rest and fix this...